corehq/apps/userreports/models.py

```diff
@@ -95,6 +95,12 @@ class Meta(object):
         # prevent JsonObject from auto-converting dates etc.
         string_conversions = ()
 
+    @classmethod
+    def wrap(self, data):
+        if not data.get('backend_id', None):
```

Comment:
also needed to add this to __init__ in https://github.com/dimagi/commcare-hq/pull/13093/commits/4cc50c728a1e6e9355aec5be3e9aaa3eb17c1144 (due to more tests I'm adding in another branch) which makes me like this even less.